package.json

```diff
@@ -134,7 +136,8 @@
     "stylelint-scss": "^3.18.0",
     "ts-loader": "^8.0.12",
     "typescript": "^4.1.3",
-    "vnu-jar": "^20.6.30"
+    "vnu-jar": "^20.6.30",
+    "webpack": "^4.44.2"
```

Comment:
Why do we need this? Isn't it the same predicament as with `eslint` or `jest` or `babel-loader`?